core/src/main/java/io/questdb/std/Files.java

```diff
@@ -178,6 +178,33 @@ public static long getDiskSize(LPSZ path) {
         return getDiskSize(0);
     }
 
+    public static long getDirectoryContentSize(Path path) {
+        int len = path.length();
+        long addr = path.address();
+        long pFind = findFirst(addr);
+        if (pFind > 0L) {
+            try {
+                long totalSize = 0L;
+                do {
+                    long namePtr = findName(pFind);
+                    Chars.utf8DecodeZ(namePtr, path.trimTo(len).slash$());
+                    path.$();
+                    if (findType(pFind) == Files.DT_FILE) {
+                        totalSize += length0(addr);
```

Comment:
this is UTF16 encoded string by `utf8DecodeZ` just above, what does `length0()` take as an arg? Perhaps argument name should be explanatory?

Reply:
renaming it to `pathPtr` for better clarity.